app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/settings/components/OptionScaffold.kt

```diff
@@ -43,14 +48,16 @@ fun OptionScaffold(
     setSelectedSettingOptionPosition: (Int) -> Unit,
     navigateBack: (() -> Unit)? = null,
 ) {
-    Scaffold(topBar = {
-        val backNavigationMenu = if (LocalNavType.current == NavigationSuiteType.NavigationBar) {
-            TopAppBarButton.backButton(navigateBack ?: {})
-        } else {
-            null
-        }
-        SwissTransferTobAppBar(topAppBarTitleRes, navigationMenu = backNavigationMenu)
-    }) { paddingsValue ->
+    PhoneTopAppBarScaffold(
+        phoneTopAppBar = {
+            val navigationMenu = if (LocalNavType.current == NavigationSuiteType.NavigationBar) {
+                TopAppBarButton.backButton(navigateBack ?: {})
+            } else {
+                null
+            }
+            SwissTransferTobAppBar(topAppBarTitleRes, navigationMenu)
```

Comment:
Do you still need to have a condition for the backButton? Because one of these cases doesn't look possible anymore